### ISSUE TITLE
Can't compile with Xcode 11 for iOS 13

### ISSUE THREAD

**Issue by wouterds** [chirag04/react-native-in-app-utils#216]
I saw this has been fixed in #211 and has been merged in master but there's no new release yet.

Installing directly from master branch works though but would prefer to have a payment lib fixed on a version rather than on the master branch. As for now I'm using this workaround.

```bash
npm i --save react-native-in-app-utils@chirag04/react-native-in-app-utils#master
```

**Comment by flikQ** [comment]
Same here, would be great to have this on a release.

In case anyone needs a yarn equivalent 

`yarn add react-native-in-app-utils@chirag04/react-native-in-app-utils#master`

**Comment by ohtangza** [comment]
I think this is specific to Xcode 13 because I am experiencing this with Xcode 13 and iOS12 simulator.

**Comment by chirag04** [comment]
published in 6.0.2

**Comment by wouterds** [comment]
Thanks @chirag04 🙏
